content/apprenticeships/Flatiron.md

```diff
@@ -0,0 +1,9 @@
+---
+company: "Flatiron Health"
+description: "In this role, you'll work with a software engineering team to help oncologists spend more time with patients or empower scientists to conduct research faster."
+image: "/images/apprenticeships/flatiron.jpeg"
+link: "https://flatiron.com/"
```

Comment:
Can you change this to a direct link to the listing or program? This is a link I found:
https://flatiron.com/careers/open-positions/2451150